src/libs/TransactionUtils.ts

```diff
@@ -631,7 +658,7 @@ function isOnHold(transaction: OnyxEntry<Transaction>): boolean {
         return false;
     }
 
-    return !!transaction.comment?.hold;
+    return !!transaction.comment?.hold || isDuplicate(transaction.transactionID, true);
 }
```

Comment:
add `isDuplicate` to condition here cause normal "Unhold" option shows for duplicated expense. and we handled it here https://github.com/Expensify/App/issues/44892